### PR TITLE
PMK-880-iDNS Weighted Policy doc update_EN

### PR DESCRIPTION
**Pull request by Luize-boyen**
documentation update
Aqui está @ederson-vieira

### CODE REVIEW

File: intelligent-dns/2021-01-14-index.md [hunk 1]
```diff
@@ -47,52 +47,112 @@ It’s designed for those that want high performance and high availability for t
 
 ---
 
-### **Creating a Hosting Zone**
+### Creating a Hosting Zone
 
-1.  Go to [Real-Time Manager](https://manager.azion.com/), access *Edge Routing > Intelligent DNS* menu. You will see the main interface where you can create and manage your zones;
-2.  Click the *Add Zone* button and fill in the following fields:  
+1.  Go to [Real-Time Manager](https://manager.azion.com/), on the **Products** menu, access **Edge Routing **> **Intelligent DNS**. You will see the main interface where you can create and manage your zones;
```
Comment (ederson-vieira) [inline intelligent-dns/2021-01-14-index.md:52]: Does the customers know where is the "Products menu"?

File: intelligent-dns/2021-01-14-index.md [hunk 2]
```diff
@@ -47,52 +47,112 @@ It’s designed for those that want high performance and high availability for t
...
+2.  Click the **Add Zone** button and fill in the following fields:  
     **Name for zone**: Identifier for the newly created zone of the domain. This field is free text. You can enter text in whatever form suits you best;
-    **Domain**: You can only enter the [Naked Domain](https://en.wikipedia.org/wiki/DNS_zone) in [FQDN](https://pt.wikipedia.org/wiki/FQDN) format, in this field. E.g.: _mydomain.com_;  
+    **Domain**: You can only enter the [Naked Domain](https://en.wikipedia.org/wiki/DNS_zone) in [FQDN](https://pt.wikipedia.org/wiki/FQDN) format, in this field. Example: _mydomain.com_;
```
Comment (ederson-vieira) [inline intelligent-dns/2021-01-14-index.md:55]: (we doesn't evaluate if it is a naked domain, but just using them prevent us some headaches)

File: intelligent-dns/2021-01-14-index.md [hunk 3]
```diff
@@ -47,52 +47,112 @@ It’s designed for those that want high performance and high availability for t
...
-    **Domain**: You can only enter the [Naked Domain](https://en.wikipedia.org/wiki/DNS_zone) in [FQDN](https://pt.wikipedia.org/wiki/FQDN) format, in this field. E.g.: _mydomain.com_;  
+    **Domain**: You can only enter the [Naked Domain](https://en.wikipedia.org/wiki/DNS_zone) in [FQDN](https://pt.wikipedia.org/wiki/FQDN) format, in this field. Example: _mydomain.com_;  
     **Active**: This flag indicates whether the zone is active and therefore whether the Intelligent DNS should respond to it or not. Deactivated zones are ignored in DNS responses;
-3.  Click the *Save* button to finish.
+3.  Click the **Save** button to finish.
 
----
+### Creating or editing records linked to a hosting zone
+
+1. Go to [Real-Time Manager](https://manager.azion.com/), access the **Products** menu,  **Edge Routing **> **Intelligent DNS**.
```
Comment (ederson-vieira) [inline intelligent-dns/2021-01-14-index.md:61]: Again, does our customers know where is the "Products menu"?

File: intelligent-dns/2021-01-14-index.md [hunk 4]
```diff
@@ -47,52 +47,112 @@ It’s designed for those that want high performance and high availability for t
...
+   | SRV - Location of server or service         | A Service record (SRV record) is a specification of data in the Domain Name System defining the location, for example: the hostname and port number, of servers for specified services. It is defined in RFC 2782, and its type code is 33. |
+   | TXT - Text                                  | A TXT record (short for text record) is a type of resource record in the Domain Name System (DNS) used to provide the ability to associate arbitrary text with a host or other name, such as human readable information about a server, network, data center, or other accounting information. |
+
+   #### Value
+
+   This is the DNS response to the registered record, such as an IPv4 address for example.<br/>[(see section - Rules for filling in values)](#rules-for-filling)
+
+   #### TTL (seconds):
+
+   This is the time a response can be cached for on a resolver server. Time-to-live (TTL) is a value in an Internet Protocol (IP) packet that tells a network router whether or not the packet has been in the network too long and should be discarded.resolver
```
Comment (ederson-vieira) [inline intelligent-dns/2021-01-14-index.md:106]: Possibly a miss-typed word at the end of the line.

File: intelligent-dns/2021-01-14-index.md [hunk 5]
```diff
@@ -47,52 +47,112 @@ It’s designed for those that want high performance and high availability for t
...
+
+   It allows you to specify which policy iDNS should consider while resolving requests answered by this record entry.
+
+   > You can select either **Simple** Policy – to use the standard DNS functionality – or  **Weighted** Policy – to specify the amount of traffic to send to each Record.
+
+   By choosing the **Weighted** Policy, there will be two more fields to be completed, which are:
+   
+   **Weight:** you can choose a number between 0 and 255 to specify the weight for each record. When you choose 0, Intelligent DNS stops using this record.
+   
+   **Description:** used to differentiate records with the same name and type. Add a description that uniquely identifies each record.
```
Comment (ederson-vieira) [inline intelligent-dns/2021-01-14-index.md:118]: This field is not unique anymore. Customers may add unique identifiers but its up to them.

File: intelligent-dns/2021-01-14-index.md [hunk 6]
```diff
@@ -47,52 +47,112 @@ It’s designed for those that want high performance and high availability for t
...
+   
+   That's it! Your domain is now set up and ready to be served by Azion's Intelligent DNS. To make sure everything is working well, run some of the tests described in the section  [Testing my zone](#test-my-zone).
+
+---
 
-Go to [Real-Time Manager](https://manager.azion.com/), access the menu *Edge Routing > Intelligent DNS*. Choose the domain you want to manage. You will see the main interface where you can create and edit your zones. When you do this, you will see two tabs:
+### Record Policies
 
-**Main Settings**
+While you are adding or modifying a record on Intelligent DNS, you can find the **“Policy“** field. This field is enabled just when you are configuring either A, AAAA, CNAME, ANAME or MX type records. This field lets you to choose between two policies to specify how Intelligent DNS should deal with requests which respond by this record.
```
Comment (ederson-vieira) [inline intelligent-dns/2021-01-14-index.md:130]: "you may find" and "answered by this record" fits better, I think.

File: intelligent-dns/2021-01-14-index.md [hunk 7]
```diff
@@ -47,52 +47,112 @@ It’s designed for those that want high performance and high availability for t
...
-| Name               | This description is the domain that you want to create. For example: "*help*", in "*help.azion.com*", or then "*manager*" in "*manager.azion.com*". |
-| Type               | This is the type of record that is being added, such as types A, AAAA, MX or CNAME, among other examples.<br/>[(see section - Types of supported entries and recommendations for each)](#types-of-entries) |
-| Value              | This is the DNS response to the registered record, such as an IPv4 address for example.<br/>[(see section - Rules for filling in values)](#rules-for-filling) |
-| TTL - Time To Live | This is the time a response can be cached for on a resolver server. |
+By selecting *weighted policy*, different fields options are displayed:
 
-> Click  the *Save* button to finish creating your record. 
+| Field                 | Description                                                  |
+| --------------------- | ------------------------------------------------------------ |
+| Weight<br>**requered* | You need to specify a weight value between 0 and 255. Higher values increases the chances of a specific record to be used. Setting 0 disables the record. |
```
Comment (ederson-vieira) [inline intelligent-dns/2021-01-14-index.md:144]: required

File: intelligent-dns/2021-01-14-index.md (hunk 1 above)
Comment (Luize-boyen) [reply intelligent-dns/2021-01-14-index.md:52]: adjusted

File: intelligent-dns/2021-01-14-index.md (hunk 3 above)
Comment (Luize-boyen) [reply intelligent-dns/2021-01-14-index.md:61]: adjusted

File: intelligent-dns/2021-01-14-index.md (hunk 7 above)
Comment (Luize-boyen) [reply intelligent-dns/2021-01-14-index.md:144]: fixed

File: intelligent-dns/2021-01-14-index.md (hunk 2 above)
Comment (ederson-vieira) [reply intelligent-dns/2021-01-14-index.md:55]: We need to replace "Naked Domain" (with its link) by Domain Name. We need to add a newline before this line too.

**Review by Luize-boyen — COMMENTED** [review]
Hi @ederson-vieira :) 
Feito os ajustes. Qq coisa me avisa que arrumo, certo?

**Review by ederson-vieira — APPROVED** [review]
Changes aproved.
Please, wait to merge only after de deployment.

File: intelligent-dns/2021-01-14-index.md [hunk 8]
```diff
@@ -29,70 +29,130 @@ permalink_pt-br: "/documentacao/produtos/intelligent-dns/"
...
-| Name               | This description is the domain that you want to create. For example: "*help*", in "*help.azion.com*", or then "*manager*" in "*manager.azion.com*". |
-| Type               | This is the type of record that is being added, such as types A, AAAA, MX or CNAME, among other examples.<br/>[(see section - Types of supported entries and recommendations for each)](#types-of-entries) |
-| Value              | This is the DNS response to the registered record, such as an IPv4 address for example.<br/>[(see section - Rules for filling in values)](#rules-for-filling) |
-| TTL - Time To Live | This is the time a response can be cached for on a resolver server. |
+By selecting *weighted policy*, different fields options are displayed:
 
-> Click  the *Save* button to finish creating your record. 
+| Field                      | Description                                                  |
+| -------------------------- | ------------------------------------------------------------ |
+| Weight<br>**required*      | You need to specify a weight value between 0 and 255. Higher values increases the chances of a specific record to be used. Setting 0 disables the record. |
```
Comment (ederson-vieira) [inline intelligent-dns/2021-01-14-index.md:144]: Por favor, troque o "of a specific" por "this specific"

File: intelligent-dns/2021-01-14-index.md (hunk 6 above)
Comment (ederson-vieira) [reply intelligent-dns/2021-01-14-index.md]: Sure, "you will find" is better.

File: intelligent-dns/2021-01-14-index.md [hunk 9]
```diff
@@ -29,70 +29,132 @@ permalink_pt-br: "/documentacao/produtos/intelligent-dns/"
...
 
 ---
 
 ## 2. How Intelligent DNS works {#how-it-works}
 
-It’s designed for those that want high performance and high availability for their hosting and their domain. Intelligent DNS is an authoritative DNS solution, and allows the client to manage their domains, zones and records through a friendly and intuitive interface. 
+It is designed for those that want high performance and high availability for their hosting and their domain. Intelligent DNS is an authoritative DNS solution, and allows the customers to manage their domains, zones and records through a friendly and intuitive interface. 
 
-> You can create ANAME type entries, which allow you to host and use *naked domains* with other Azion products.
+> You can create ANAME type entries, which allow you to host and use your *domains* with other Azion products.
```
Comment (ederson-vieira) [inline intelligent-dns/2021-01-14-index.md:42]: Here, we need to use "naked domains".

**Pull request merged**